packages/arb-token-bridge-ui/src/token-bridge-sdk/CctpTransferStarter.ts

```diff
@@ -80,6 +74,15 @@ export class CctpTransferStarter extends BridgeTransferStarter {
     )
   }
 
+  public async transferEstimateGas() {
+    // for cctp transfers we don't call our native gas estimation methods because we have completely different contracts
+    return {
+      estimatedL1Gas: constants.Zero,
+      estimatedL2Gas: constants.Zero,
+      estimatedL2SubmissionCost: constants.Zero
+    }
```

Comment:
Should we just return `undefined` in this case? That seems better, then the UI can handle that case.